decimal.go

```diff
@@ -447,6 +447,26 @@ func (d Decimal) MarshalText() (text []byte, err error) {
 	return []byte(d.String()), nil
 }
 
+func (d Decimal) Min(others ...Decimal) Decimal {
```

Comment:
I would prefer these to be functions, not methods. It looks much better to do `decimal.Min(a, b, c)` than to do `a.Min(b, c)`